dockers/cuda-nightly/Dockerfile

```diff
@@ -0,0 +1,65 @@
+ARG CUDNN_VERSION=7
+ARG CUDA_VERSION=10.1
+
+FROM nvidia/cuda:${CUDA_VERSION}-cudnn${CUDNN_VERSION}-devel
```

Comment:
Is this the correct image for installing various PyTorch versions?

Reply:
@Borda